chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -721,3 +721,28 @@ mac-word-plugin-install-dont-ask-again-button =
     .label = { general-dont-ask-again }
 
 connector-version-warning = The { -app-name } Connector must be updated to work with this version of { -app-name }.
+
+tabs-menu-row =
+    .aria-description = You are currently on tab { $index } of { $total }.
+    To search for tabs, type the keyword. To jump to the currently selected tab, press Enter.
```

Comment:
- Has to be `{ return-or-enter }` to account for macOS
- "keyword" seems a little unclear. "To search for a tab, type part of its name."? (Below too.)